projects/schematics/jest.schematics.config.js

```diff
@@ -0,0 +1,28 @@
+const { pathsToModuleNameMapper } = require('ts-jest/utils');
+const { compilerOptions } = require('./tsconfig.schematics.spec');
+
+module.exports = {
+  setupFilesAfterEnv: ['<rootDir>/jest.ts'],
+  transform: {
+    '^.+\\.ts?$': 'ts-jest',
+  },
+
+  collectCoverage: true,
```

Comment:
Can we have coverage disabled by default and only use it in CI with `--coverage` flag?

Reply:
Sure, done.